{{cookiecutter.pipeline_slug}}/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: nfcore-{{ cookiecutter.pipeline_name }}
+dependencies:
+  - fastqc=0.11.5
```

Comment:
For bioconda specific reasons, FastQC needs openjdk installed too, see https://github.com/ewels/NGI-RNAseq/blob/2b4f5cbe2f87cbbdcccb323d4387d28798ceaf76/environment.yml#L10-L11

```yaml
  - conda-forge::openjdk=8.0.144 # Needed for FastQC docker - see bioconda/bioconda-recipes#5026
  - fastqc=0.11.7
```